src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -1,44 +1,62 @@
 import * as React from 'react';
 import {Dimensions, Keyboard, Pressable, ScrollView, StyleSheet, Text, View} from 'react-native';
 
+import {useRoute} from '@react-navigation/core';
 import {colors} from '../../../utils/colors';
 import {fonts, normalizeFontSize} from '../../../utils/fonts';
+import {setNavbarTitle} from '../../../context/actions/setMyProfileAction';
+import {Context} from '../../../context';
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements are not properly formatted. They should be grouped and ordered according to the type of module (built-in, external, internal). Also, there is an unnecessary space before some import statements.
```diff
- import {useRoute} from '@react-navigation/core';
-  import {colors} from '../../../utils/colors';
-  import {fonts, normalizeFontSize} from '../../../utils/fonts';
- import {setNavbarTitle} from '../../../context/actions/setMyProfileAction';
- import {Context} from '../../../context';

+ import * as React from 'react';
+ import {Dimensions, Keyboard, Pressable, ScrollView, StyleSheet, Text, View} from 'react-native';
+ import {useRoute} from '@react-navigation/core';
+ import {colors} from '../../../utils/colors';
+ import {fonts, normalizeFontSize} from '../../../utils/fonts';
+ import {setNavbarTitle} from '../../../context/actions/setMyProfileAction';
+ import {Context} from '../../../context';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->